### ISSUE TITLE
edX Intro to Web A11y course page is missing a logo

### ISSUE THREAD

**Issue by wittjeff** [Archive-42/W3-Docs-Website#2]
Bug on course page: https://www.edx.org/learn/web-accessibility/the-world-wide-web-consortium-w3c-introduction-to-web-accessibility
Under More about this course, UNESCO iite logos are missing.  
Path is https://prod-edx-mktg-edit.edx.org/sites/default/files/logo-unesco-iite.png
You seem to use that file in other locations as well.
